### DISCUSSION TITLE
Module M and V tests fail after Installation

### DISCUSSION THREAD

**Discussion by montag-0506** [neperfepx/neper#674] [Q&A] [answered]
Hello everyone,

If tried installing Neper multiple times now and 223 of 380 tests keep failing.

I tried following guides:
https://stackoverflow.com/questions/52594533/neper-software-installation
https://neper.info/doc/introduction.html#installing-neper
https://www.youtube.com/watch?v=8reOruZaRUA

I tried to install it in multiple VMs, WSL and on a physical PC.
I'm running Ubuntu 22.04 on all the machines.

Can someone please help me?

Greetings,
Lukas

**Comment by montag-0506** [comment]
I'm down to V/space_pf_* tests failing after installing texlive and asymptome

**Comment by montiMech** [comment]
Hi, I have similiar issues, mabe we can figure it out together. I am using a docker image of ubuntu 20.04 running the following commands: 

```
sudo apt update -y && \
    sudo apt install -y \
    build-essential \
    libpthread-stubs0-dev \
    libgsl-dev \
    libomp-dev \
    libboost-thread-dev \
    povray \
    asymptote \
    vim \
    cmake \
    tar \
    git \
    wget

#---Download and extract Gmsh sourcecode from---
#   https://gmsh.info/
wget -O gmsh-4.11.1-source.tgz "https://gmsh.info/src/gmsh-4.11.1-source.tgz"
tar -xf gmsh-4.11.1-source.tgz

git clone https://github.com/rquey/neper.git

cd /home/gmsh-4.11.1
mkdir build
cd build
cmake ..
make
make test
sudo make install

export FORCE_BUILTIN_NLOPT=ON
export FORCE_BUILTIN_LIBSCOTCH=ON

cd /home/neper/src
mkdir build
cd build
cmake ..
make
make test
sudo make install
```

all gmsh tests pass. For Neper it is sometimes the case that test 166 wont pass and the tests V/space_pf_* never pass.

**Comment by montag-0506** [comment]
I'm down to V/asymptote and V/povray failing even though both are installed.

**Comment by montag-0506** [comment]
I'm down to V/asymptote failing. Problem with povray was the version

**Comment by rquey** [comment, marked as answer]
Please follow the instructions from the [documentation](https://neper.info/doc/introduction.html#installing-neper) - the other resources are outdated or unofficial.

Should you have any difficulties with one of the steps, please provide detailed information, so that we can 1/ help you and 2/ improve the documentation as necessary.
